hedera-node/src/main/java/com/hedera/services/tokens/HederaTokenStore.java

```diff
@@ -545,6 +539,20 @@ private ResponseCodeEnum symbolCheck(String symbol) {
 				: INVALID_TOKEN_SYMBOL;
 	}
 
+
+	private ResponseCodeEnum tokenNameCheck(String tokenName) {
+		if (tokenNameKeyedIds.containsKey(tokenName)) {
+			return TOKEN_NAME_ALREADY_IN_USE;
+		}
+		if (tokenName.length() < 1) {
+			return MISSING_TOKEN_NAME;
+		}
+		if (tokenName.length() > 100) {
```

Comment:
I think that if you set the properties as per my previous comments you will be able to get the global variable here. Nevertheless, this should not be hardcoded